src/ripple/protocol/impl/Feature.cpp

```diff
@@ -146,7 +145,6 @@ uint256 const featureTickets = *getRegisteredFeature("Tickets");
 uint256 const featureTrustSetAuth = *getRegisteredFeature("TrustSetAuth");
 uint256 const featureOwnerPaysFee = *getRegisteredFeature("OwnerPaysFee");
 uint256 const featureCompareFlowV1V2 = *getRegisteredFeature("CompareFlowV1V2");
-uint256 const featureSHAMapV2 = *getRegisteredFeature("SHAMapV2");
```

Comment:
I appreciate the cautious approach. Amendments are, in a sense, orthogonal to server versions. A server can either support a given amendment or it may not. It's perfectly fine to remove an amendment that was previously introduced but which hasn't been activated; it may even be good to de-clutter the code.